api/src/main/java/org/openmrs/Concept.java

```diff
@@ -1419,8 +1419,9 @@ public String toString() {
 			}
 		}
 		catch (Exception e) {
-			// pass
+		 logger.error("Error", e);
```

Comment:
The reason CI is failing is because the variable for the `Logger` instance is `log`, as defined at the very start of the class, not `logger`.

Reply:
ok thanks @PermissionError